src/components/form-textarea/form-textarea.tsx

```diff
@@ -0,0 +1,39 @@
+import { Controller, useFormContext } from "react-hook-form";
+import { Input, Form } from "antd";
+import { FormTextareaProps } from "./form-textarea.types";
+
+const FormTextArea = ({ name, label, required, ...props }: FormTextareaProps) => {
+    const {
+        control,
+        formState: { errors, defaultValues },
+    } = useFormContext();
+
+    const errorMessage = errors[name]?.message;
+    const defaultValue = defaultValues?.[name] as string;
+
```

Comment:
Maybe should replace these evaluations with the ones used in other inputs (for the sake of consistency)

Reply:
I guess I've written the reusable text area before you made those changes, ok I think it's a good approach, I'll refactor